jms-core/src/main/java/io/micronaut/jms/configuration/JMSTopicListenerMethodProcessor.java

```diff
@@ -46,9 +46,13 @@ public JMSTopicListenerMethodProcessor(BeanContext beanContext,
 
     @Override
     protected ExecutorService getExecutorService(AnnotationValue<Topic> value) {
-
         final Optional<String> executorName = value.stringValue("executor");
-        if (executorName.isPresent() && !executorName.get().isEmpty()) {
+
+        if (!executorName.isPresent()) {
+            return null;
+        }
+
+        if (!executorName.get().isEmpty()) {
```

Comment:
why do we still need this code if the members were deprecated?

Reply:
it is to allow backward compatibility for the usage of the annotation concurency options before we completely remove concurrency support. If the annotation concurency options are configured, the implementation is utilized as before. If options are not present, the will not be conurency utilization at jms meesagee proccessing level. 

Do you want to say that we should remove the concurency utilization regardles of presence of the deprecated anotation concurency options?

Reply:
If we retain the behaviour we should as a minimum log a deprecation warning when it is used